mojaloop-ttk-simulators/chart-sim1/values.yaml

```diff
@@ -106,7 +106,34 @@ ml-testing-toolkit:
   #         host: ml-testing-toolkit-specapi.local
   #       adminApi:
   #         host: ml-testing-toolkit.local
-  #   extraEnvironments:
+    extraEnvironments:
```

Comment:
@sri-miriyala please confirm this is the same as https://github.com/mojaloop/deploy-config/blob/deploy/PI20.1/mojaloop/test.mojaloop.live/moja4-sdk-ttk-sims/bulk-ttk-moja4-env.json